### PR TITLE
Disable prelu_op_test until fixing Python3 issues

### PR DESCRIPTION
**Pull request by velconia**
this for #12835

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]

File: python/paddle/fluid/tests/unittests/test_prelu_op.py [hunk 1]
```diff
@@ -51,30 +51,28 @@ def initTestCase(self):
     def test_check_output(self):
         self.check_output()
 
-    def test_check_grad(self):
```
Comment (typhoonzero) [inline python/paddle/fluid/tests/unittests/test_prelu_op.py:54]: Why remove this test?

Comment (velconia) [reply python/paddle/fluid/tests/unittests/test_prelu_op.py:54]: Github的显示问题...这个test没被删掉, 只是下面改名成 ```test_check_grad_2``` 了..

**Comment by panyx0718** [comment]
change the PR's name if it doesn't disable test?

**Review by typhoonzero — APPROVED** [review]
LGTM++

**Comment by velconia** [comment]
It disables the preluop test

**Pull request merged**